nixos/tests/k3s/auto-deploy-charts.nix

```
@@ -78,6 +78,10 @@ import ../make-test-python.nix (
             disabled = testChart // {
               enable = false;
             };
+            # chart with values set via YAML file
+            values-file = testChart // {
+              values = ./k3s-test-chart-values.yaml;
```

As it is a very short file, would be nice to keep the test file auto-contained.

```suggestion
            values = pkgs.writeText "k3s-test-chart-values.yaml" ''
              runCommand: "echo 'Hello, file!'"
              image:
                repository: test.local/test
                tag: local
            '';
```

Since it has been 2 weeks, please remember to rebase PR to master.